src/amo/components/Addon/index.js

```diff
@@ -154,42 +160,35 @@ export class AddonBase extends React.Component {
           ? previewHeader.image_url
           : null;
 
-      const label = addon
-        ? i18n.sprintf(i18n.gettext('Preview of %(title)s'), {
-            title: addon.name,
-          })
-        : '';
-
-      const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
-
-      if (!previewURL && type === ADDON_TYPE_THEME) {
+      if (!previewURL && addon.type === ADDON_TYPE_THEME) {
         previewURL = addon.previewURL;
       }
 
-      const headerImage = (
-        <img
-          alt={label}
-          className="Addon-theme-header-image"
-          src={previewURL}
-        />
-      );
+      const themeClassName = 'Addon-theme-header';
 
       return (
         <div
-          className="Addon-theme-header"
-          id="Addon-theme-header"
+          key="Addon-theme-image"
+          className={themeClassName}
+          id={themeClassName}
```

Comment:
Do we really need an `id` here? I find it strange to use the `themeClassName`. If the `id` is not useful, let's remove it and also remove the `themeClassName`.

Reply:
Yeh I guess I just left it b/c it was there already. Sure I can remove this